src/drafts/InlineAutocomplete/InlineAutocomplete.tsx

```diff
@@ -26,6 +26,11 @@ export type InlineAutocompleteProps = {
    * `suggestions` prop accordingly.
    */
   onShowSuggestions: (event: ShowSuggestionsEvent) => void
+
+  /** Called when a suggestion is selected.
+   */
+  onSelectSuggestion?: (suggestion: string) => void
```

Comment:
To keep consistent with the rest of the props, maybe we should define a `SelectSuggestionsEvent` type and pass that instead of just the suggestion string. It could also include the trigger & the query text. This could give more useful data to the handler. For example:

```tsx
type SelectSuggestionsEvent = ShowSuggestionsEvent & {
  /** The suggestion that was selected. */
  suggestion: Suggestion
}
```

Reply:
Done. Please check commit 5d1b9fadbecb7c3f7deb64cb812cecfa0999886b